hw10-spring-Rest/library-app/src/components/book/BookList.js

```diff
@@ -0,0 +1,90 @@
+import 'bootstrap/dist/css/bootstrap.min.css';
+
+import React from 'react'
+import Card from "react-bootstrap/Card";
+import Accordion from "react-bootstrap/Accordion";
+import Button from "react-bootstrap/Button";
+import Book from "./Book";
+import {deleteBook, loadAllBooks} from "../../common/ApiServiceBooks";
+import {loadAllAuthors} from "../../common/ApiServiceAuthors";
+import {loadAllGenres} from "../../common/ApiServiceGenres";
+
+
+export default class BookList extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            eventKeyAccording: props.eventKeyAccording,
+            title: props.title,
+            books: [],
+            genres: [],
+            authors: []
+        };
+
+        this.booksListToggle = this.booksListToggle.bind(this);
+        this.addBook = this.addBook.bind(this);
```

Comment:
https://medium.com/@jacobworrel/babels-transform-class-properties-plugin-how-it-works-and-what-it-means-for-your-react-apps-6983539ffc22